Repositories/Contents.md

```diff
@@ -65,3 +66,109 @@ GET /repos/:username/:reponame/archive/:ref:format
 Status: 200 OK
 Content-Type: application/octet-stream
 ```
+
```

Comment:
I think this addition should happen in this file since this endpoint deals with git contents. Not sure if we should be keeping the already existing doc for raw and archive in the same file or move it to another file/folder.